### DISCUSSION TITLE
Alternatives to generate_ode_function

### DISCUSSION THREAD

**Discussion by Simone2021space** [sympy/sympy#22847] [Q&A]
Hi,
I noticed that generate_ode_function doesn't work when using a nested Piecewise function (which contains an ITE statement).
Are there alternatives to it, like lambdify? How to use it?

**Comment by oscarbenjamin** [comment]
What is `generate_ode_function`?

**Comment by moorepants** [comment]
This is the forum for SymPy, not PyDy. You can ask PyDy specific questions on the PyDy mailing list and the PyDy github repository.
